examples/body-to-query-params-server/app.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+app.use(bodyParser.json())
+const axios = require('axios');
+const { json } = require('stream/consumers');
+
+const port = 3000
+
+app.post('/', (req, res) => {
+    const headerOutboundUrl = req.get('Outbound-Url');
+    console.log('Outbount Url: ' + process.env.OUTBOUND_URL);
+    const outboundUrl = headerOutboundUrl + "?";
+
+    const outboundQueryParams = req.get('Query-Param-Fields');
+    console.log('Query Parameter Fields: ' + outboundQueryParams);
+
+    const params = outboundQueryParams.split(',');
+
+    console.log('POST body: ', req.body);
+
+    let outboundUrlThisRequest = outboundUrl;
+    params.forEach(param => {
+        outboundUrlThisRequest = outboundUrlThisRequest + param + '=' + req.body[param] + '&';
```

Comment:
Use the URL api here too

Reply:
also, we should handle the case if the key is not found. if i have 
```js
{
  foo: "hello",
  bar: "friend"
}
```

and i set `Query-Param-Fields: bar,baz`, should we add `&baz=`? The URL api should mostly handle this but we should likely remove any empty query params unless they explicitly want that

Reply:
I think this depends on the usecase so users of this example can change it if they like. For now we will do what the Node API does by default.